docs/user/admin/settings.rst

```diff
@@ -305,7 +305,7 @@ SQL query::
     {
       "error": {
         "reason": "Invalid SQL query",
-        "details": "DELETE clause is disabled by default and will be deprecated. Using the plugins.sql.delete.enabled setting to enable it",
+        "details": "Failed to parse query due to offending symbol [DELETE] at: 'DELETE' <--- HERE... More details: Expecting tokens in {<EOF>, 'DESCRIBE', 'SELECT', 'SHOW', ';'}\nQuery failed on both V1 and V2 SQL engines. V1 SQL engine error following: \nDELETE clause is disabled by default and will be deprecated. Using the plugins.sql.delete.enabled setting to enable it",
```

Comment:
any reason we want to expose V1/V2 engine info for customer?

Reply:
It is already exposed, see [#830](https://github.com/opensearch-project/documentation-website/pull/830)/[#1111](https://github.com/opensearch-project/documentation-website/pull/1111).
Any ideas how to do this better?
I placed V2 error first, because it is more relevant for most cases (except this one).

Reply:
as discussed, we may want to simplify error message exposed to customer and log this kind of detailed message for our developer use.

Reply:
Fixed in de104bfb.